### PR TITLE
Introducing Ads

### PR DESCRIPTION
**Pull request by andrin-n-dream**

### CODE REVIEW

File: airconsole-1.4.0.js [hunk 1]
```diff
@@ -0,0 +1,819 @@
...
+ * developers@airconsole.com. For development purposes, localhost is always
+ * allowed.
+ * @abstract
+ * @param {string|undefined} email_address - The email address of the user if
+ *        it was set.
+ */
+AirConsole.prototype.onEmailAddress = function(email_address) {};
+
+/**
+ * Gets called every X milliseconds with device motion data iff the
```
Comment (francois-n-dream) [inline airconsole-1.4.0.js]: iff = if

File: airconsole-1.4.0.js [hunk 2]
```diff
@@ -0,0 +1,819 @@
...
+ * Gets called if a fullscreen advertisement is shown on this screen.
+ * In case this event gets called, please mute all sounds.
+ * @abstract
+ */
+AirConsole.prototype.onAdShow = function() {};
+
+/**
+ * Gets called when an advertisement is finished or no advertisement was shown.
+ * @abstract
+ * @param {boolean} ad_was_shown - True iff an ad was shown and onAdShow was
```
Comment (francois-n-dream) [inline airconsole-1.4.0.js]: iff = if

File: airconsole-1.4.0.js (hunk 1 above)
Comment (andrin-n-dream) [inline airconsole-1.4.0.js]: iff = if and only if

File: airconsole-1.4.0.js (hunk 2 above)
Comment (andrin-n-dream) [inline airconsole-1.4.0.js]: iff = if and only if

**Pull request merged**